theano/tensor/tests/test_basic.py

```diff
@@ -7118,6 +7118,16 @@ def test_memmap(self):
         new_inp[...] = inp
         x = as_tensor_variable(new_inp)
 
+    def test_empty_dtype(self):
+        old = theano.config.floatX
+        for dtype in ['float16', 'float32', 'float64']:
+            try:
```

Comment:
It is only a decorator in Theano right now. I don't want to wait for the RNN PR to be able to use the context manager. My first version was with change_flags as a context manager as I taught it was merged.